formbricks/formbricks
# feat: Add color scheme for light and dark themes

## What does this PR do?

This PR introduces a new color scheme for light and dark themes in the Tailwind CSS configuration. It follows a structured naming convention, ranging from 50 to 900 shades for each color group. The goal is to improve the recognition and usability of colors in the project.

Fixes #1342 

## Type of change


## How should this be tested?

- Verify that the new color scheme is applied correctly to the UI components.
- Check if the color shades are easily distinguishable in both light and dark themes.

## Checklist

### Required

- [] First PR at Formbricks? [Please sign the CLA!](https://formbricks.com/clmyhzfrymr4ko00hycsg1tvx) Without it, we won't be able to merge it 🙏

### Appreciated
